# Pertemuan 3

## Input/Output

### Contoh Input

In [1]:
# Contoh Input String
nama = input("Masukkan nama Anda: ")
print("Halo,", nama)

# Contoh Input Integer
umur = int(input("Masukkan umur bulat: "))
print("Umur Anda:", umur)

# Contoh Input Float
ipk = float(input("Masukkan bilangan ipk: "))
print("IPK Anda:", ipk)

Halo, Budi
Umur Anda: 17
IPK Anda: 3.8


### Contoh Input Yang di Pisahkan Spasi

In [2]:
bil = input("Masukkan dua bilangan bulat dipisahkan oleh spasi: ")
# Membagi variable bil menjadi variable bil1 dan bil2
bil1, bil2 = map(int, bil.split())
print("Bilangan pertama:", bil1)
print("Bilangan kedua:", bil2)

Bilangan pertama: 1
Bilangan kedua: 3


### Macam-Macam Jenis Output Pada Python

In [3]:
print("Hallo", nama, "Umur anda:", umur, "IPK anda:", ipk)
print("Hallo "+ nama+ ", Umur anda:", umur, "IPK anda:", ipk)
print("Hallo %s, Umur anda: %i, IPK anda: %.1f" % (nama, umur, ipk))
print(f"Hallo {nama}, Umur anda: {umur}, IPK anda: {ipk}")

Hallo Budi Umur anda: 17 IPK anda: 3.8
Hallo Budi, Umur anda: 17 IPK anda: 3.8
Hallo Budi, Umur anda: 17, IPK anda: 3.8
Hallo Budi, Umur anda: 17, IPK anda: 3.8


## Struktur Runtunan

### Contoh Menghitung Luas dan Keliling Lingkaran

In [8]:
# Input
pi = 22/7
r = int(input("jari-jari = "))

# Proses menghitung luas lingkaran
luas = pi * r * r

# output luas lingkaran
print("Pi = %.2f" %pi)
print("jari-jari =", r)
print("Luas lingkaran = %.2f" %(luas))

Pi = 3.14
jari-jari = 3
Luas lingkaran = 28.29


Mendeklarasikan variable 'pi' yang digunakan untuk menyimpan nilai pi yaitu 22/7 atau 3.14.
Mendeklarasikan variable 'r' yang digunakan untuk menyimpan inputan dari user, dengan tipe data integer.
Mendeklarasikan variable 'luas' yang digunakan untuk menghitung luas lingkaran.
'print("Pi = %.2f" %pi)' digunakan untuk menampilkan nilai atau isi dari variable 'pi', dan '%.2f' digunakan untuk menampilkan menampilkan 2 bilangan setelah (,).
'print("jari-jari =", r)' digunakan untuk menampilkan nilai atau isi dari variable 'r'.
'print("Luas lingkaran = %.2f" %(luas))' digunakan untuk menampilkan nilai atau isi dari variable luas, dan '%.2f' digunakan untuk menampilkan menampilkan 2 bilangan setelah (,).

In [9]:
# Proses menghitung keliling lingkaran
keliling = 2 * pi * r

# output keliling lingkaran
print("Pi = %.2f" %pi)
print("jari-jari = ", r)
print("Keliling lingkaran = %.2f" %(keliling))

Pi = 3.14
jari-jari =  3
Keliling lingkaran = 18.86


Mendeklarasikan variable 'keliling' yang digunakan untuk menghitung keliling lingkaran.
'print("Pi = %.2f" %pi)' digunakan untuk menampilkan nilai atau isi dari variable 'pi', dan '%.2f' digunakan untuk menampilkan menampilkan 2 bilangan setelah (,).
'print("jari-jari =", r)' digunakan untuk menampilkan nilai atau isi dari variable 'r'.
'print("Luas lingkaran = %.2f" %(luas))' digunakan untuk menampilkan nilai atau isi dari variable luas, dan '%.2f' digunakan untuk menampilkan menampilkan 2 bilangan setelah (,).